### DISCUSSION TITLE
PyMISP import with XML file

### DISCUSSION THREAD

**Discussion by swebster96** [MISP/PyMISP#840] [Q&A] [answered]
All examples in PyMISP for importing data are either CSV of JSON.   I created export files in XML from a database that need to be imported in MISP as Events.   The XML file imports correctly through the application under Import... but this needs to be automated.   I reviewed the examples and have not found an import method for XML using Python3.   Suggestions?

**Comment by Rafiot** [comment, marked as answer]
Please use JSON for import/export, XML isn't supported and shouldn't be used anymore.

**Reply by swebster96** [reply]
Okay, thanks for the reply.  
In terms of uploading multiple Events from a JSON file, is there existing example code or python scripts to test and play with in PyMISP?
